### ISSUE TITLE
see transaction for each account separately

### ISSUE THREAD

**Issue by tunwa** [mvarnagiris/financius#380]
Able to filter/ When click into each account should have option to be able to see transaction for each account individually, instead of see

**Comment by AFirooz** [comment]
Yes please

**Comment by preno** [comment]
Came here to say this, currently using multiple accounts are 100% useless, since we can't filter anyway.

**Comment by manwz** [comment]
Yes, please. Add this section in the account panel, just below the information that is right now there.

**Comment by thisbejohn** [comment]
This feature that would keep me using this app for a very long time.

**Comment by shashwatblack** [comment]
This would be very very welcome.
